subprojects/docs/src/docs/userguide/java_plugin.adoc

```diff
@@ -488,50 +482,48 @@ Default value: `__docsDir__/javadoc`
 Default value: The name and version of the project
 
 [[sec:clean]]
-=== Clean
+== Clean
 
-The `clean` task is an instance of api:org.gradle.api.tasks.Delete[]. It simply removes the directory denoted by its `dir` property.
+The `clean` task is an instance of link:{groovyDslPath}/org.gradle.api.tasks.Delete.html[Delete]. It simply removes the directory denoted by its `dir` property.
 
-==== Clean properties
+=== Clean properties
 
 `File dir`::
 Default value: `__buildDir__`
 
 
-[[sec:java_resources]]
-=== Resources
+== Resources
 
-The Java plugin uses the api:org.gradle.api.tasks.Copy[] task for resource handling. It adds an instance for each source set in the project. You can find out more about the copy task in <<sec:copying_files>>.
+The Java plugin uses the link:{groovyDslPath}/org.gradle.api.tasks.Copy.html[Copy] task for resource handling. It adds an instance for each source set in the project. You can find out more about the copy task in <<working_with_files.adoc#sec:copying_files,Copying files>>.
```

Comment:
In old doc, `<<working_with_files.adoc#sec:copying_files,Copying files>>` is rendered as subsection title `“File copying in depth”`, but in new doc this seems to be missing.

Reply:
Changed to "File copying in depth"